app/Models/Zhanr.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Zhanr extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'nazvanie',
```

Comment:
транслит)))

```suggestion
        'name',
```
или
```suggestion
        'title',
```